tests/Microsoft.Identity.Test.Integration/Infrastructure/SeleniumWebUI.cs

```diff
@@ -25,28 +27,31 @@ internal class SeleniumWebUI : IWebUI
   <head><title>Authentication Failed</title></head>
   <body>
     Authentication failed. You can return to the application. Feel free to close this browser tab.
+</br></br></br></br>
+    Error details: error {0} error_description: {1}
```

Comment:
I'm not sure this would apply here, but something I've seen being really nice in Azure CLI (which uses a similar technics) is after a few seconds of displaying a message, the page redirects to their doc (the Azure CLI doc)
Not sure if we'd want here to redirect to the wiki, or to the build status dashboard (even if it's not purely related to running this test)

Reply:
This part is not really needed, as there is no user involved. I chose to include it just to pre-test system browser auth on desktop, see how it will behave when automated. But this is a good idea for productizing the feature.

During test, the browser is hidden.